apps/modernization-ui/src/apps/search/patient/PatientCriteria/Address.spec.tsx

```diff
@@ -0,0 +1,31 @@
+import { render } from '@testing-library/react';
+import { Address } from './Address';
+import { PatientCriteriaEntry } from '../criteria';
+import { FormProvider, useForm } from 'react-hook-form';
+import { renderHook } from '@testing-library/react-hooks';
+import { MockedProvider } from '@apollo/react-testing';
+
+const { result } = renderHook(() =>
+    useForm<PatientCriteriaEntry>({
+        mode: 'onChange',
+        defaultValues: { status: [{ name: 'Active', label: 'Active', value: 'ACTIVE' }] }
+    })
+);
+
```

Comment:
suggestion (non-blocking):  You only have to use `renderHook` when you are testing hooks, the `useForm` can be initialized in the `setup` function.